test/fixtures/models/katello_root_repositories.yml

```diff
@@ -150,6 +150,17 @@ busybox2_root:
   url:                  'http://docker.io'
   docker_upstream_name: 'busybox'
 
+ssh_root:
+  name:                 ssh
+  content_id:           1
+  content_type:         docker
+  label:                ssh
+  product_id:           <%= ActiveRecord::FixtureSet.identify(:puppet_product) %>
+  gpg_key_id:           <%= ActiveRecord::FixtureSet.identify(:fedora_gpg_key) %>
+  unprotected:           <%= true %>
+  url:                  'https://registry-1.docker.io'
+  docker_upstream_name: 'fedora/ssh'
```

Comment:
lets not add another docker repo to fixtures.  Lets try to adapt one that already exists if what we have isn't sufficient.